themes/src/main/resources/theme/keycloak.v2/account/src/app/content/device-activity-page/DeviceActivityPage.tsx

```diff
@@ -261,14 +261,14 @@ export class DeviceActivityPage extends React.Component<DeviceActivityPageProps,
                           <React.Fragment key={'device-' + deviceIndex + '-session-' + sessionIndex}>
                             <DataListItemRow>
                               <DataListContent aria-label="device-sessions-content" isHidden={false} className="pf-u-flex-grow-1">
-                                <Grid className="signed-in-device-grid" hasGutter>
+                                <Grid id={session.id} className="signed-in-device-grid" hasGutter>
```

Comment:
@ssilvert @mposolda I'm not sure whether the full session ID can be included here due to some security risks? Are there any? The session ID is visible in some REST calls as a Path parameter, so the attacker can eavesdrop the session ID anyway. However, I don't think it's a security issue. I just need to have it verified. Thanks

Reply:
Good question. My vote is to rather safe than sorry and not include session.id . Or is there some good reason why is it needed? In case that "Grid" needs unique id for each session, is it an option to use for example simple counter (EG. "session-1, "session-2" etc?)

Reply:
@mposolda Thanks for your comment. Agree, it'd be better to edit only those tests.